Packs/ComputerVisionEngine/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "ComputerVisionEngine",
```

Comment:
```suggestion
    "name": "Computer Vision Engine",
```

Reply:
Okay. Do i need to change the name in the yml file too ?

Reply:
fixed